package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1286,6 +1286,82 @@ class AltLocs(AtomAttr):
     def _gen_initial_values(na, nr, ns):
         return np.array(['' for _ in range(na)], dtype=object)
 
+class GBScreens(AtomAttr):
```

Comment:
Being able to add all these attributes shows off the flexibility of the topology system!